docs/release-notes.rst

```diff
@@ -5,7 +5,7 @@ Release Notes
 *************
 Version 0.6.4
 *************
-Release Date xx/xx/24
+Release Date 31/01/25
```

Comment:
Did you mean "13" and not "31"

Reply:
In my head I was thinking we would have a bunch of dependency updates that I would be sifting through, so I set the date to EOM, and the PR to draft. 
Doesn't look like there are any updates, so I think we are good to go.